.env

```
@@ -4,13 +4,19 @@ MYSQL_USER=prestashop
 MYSQL_PASSWORD=prestashop
 
 #PS Versions to use
-PS_VERSION=1.7.8.9
-PHP_VERSION=7.4
+PS_VERSION=1.6
+PHP_VERSION=5.6
```

Creo que sería mejor dejar las versiones que tenemos por defecto:

```suggestion
PS_VERSION=1.7.8.9
PHP_VERSION=7.4
```

---

Okey!